queryfilter/queryfilter.py

```diff
@@ -1,50 +1,55 @@
 from __future__ import absolute_import
 
 from .base import FieldFilter
-from .utils import is_same_class
 
 
 class QueryFilter(object):
-    filter_candidates = {}
+    filters_mapping = {}
 
     @classmethod
-    def get_filter_key_from_type_and_condition(cls, filter_type,
-                                               filter_condition):
+    def get_filter_key(cls, filter_type, filter_condition):
         return "{}.{}".format(filter_type, filter_condition)
 
     @classmethod
     def register_type_condition(cls, filter_type, filter_condition=None):
+        def _is_same_class(x_cls, y_cls):
```

Comment:
when `def` inside a function it's not not need of `_` prefix, like `def is_same_class`